model/src/main/java/org/openremote/model/datapoint/AssetDatapointResource.java

```diff
@@ -42,7 +40,7 @@ public interface AssetDatapointResource {
      * not have datapoint storage enabled.
      */
     @POST
-    @Path("{assetId}/attribute/{attributeName}")
+    @Path("{assetId}/{attributeName}")
```

Comment:
For consistency in endpoint paths I would either;
- rename the `POST` and `PUT` endpoints in `AssetPredictedDatapointResource` to this one.
- or revert this to `{assetId}/attribute/{attributeName}`

Reply:
Good point I've updated `AssetPredictedDatapointResource` to match